v1/client.go

```diff
@@ -60,6 +60,12 @@ type Client struct {
 
 // DefaultClient returns a client that connects to the Atlas API.
 func DefaultClient() *Client {
+	atlasEndpoint := os.Getenv("ATLAS_ADDRESS")
```

Comment:
Should probably make `ATLAS_ADDRESS` a constant, and I think we should rename it to `ATLAS_ADDR` to be more in line with our other projects (Consul, Vault, Nomad)